apps/minifront/src/state/dutch-auction/index.ts

```diff
@@ -135,4 +148,20 @@ export const createDutchAuctionSlice = (): SliceCreator<DutchAuctionSlice> => (s
   },
 
   metadataByAssetId: {},
+
+  endAuction: async auctionId => {
+    set(state => {
+      state.dutchAuction.txInProgress = true;
+    });
+
+    try {
+      const req = new TransactionPlannerRequest({ dutchAuctionEndActions: [{ auctionId }] });
```

Comment:
No `assemble*Request()` helper needed here, because creating the request is so simple. 

However, I still renamed the other helper to `assembleScheduleRequest()` in this PR to make it clear which request type is being assembled.